backend/endpoints/models/volume_models.go

```diff
@@ -32,6 +32,6 @@ type (
 type (
 	// DocumentRetrievalResponse is just the returned response for any handler that fetches the contents of a docker volume
 	DocumentRetrievalResponse struct {
```

Comment:
Do we still need this type? The output type of `GetPublishedDocument` is now just an array of bytes as opposed to a nicely formatted json response

Reply:
Good catch removed now